Tasks/cURLUploader/task.json

```diff
@@ -0,0 +1,78 @@
+{
+    "id": "AD8974D8-DE11-11E4-B2FE-7FB898A745F3",
+    "name": "cURLUploader",
+    "friendlyName": "cURL Upload Files",
+    "description": "Use cURL to upload files with supported protocols. (FTP, FTPS, SFTP, HTTP, and more)",
+    "category": "Utility",
```

Comment:
I think this task is on the borderline and it makes sense for it to be in either "utility" or "deploy".  This was in deploy category before, but I really feel this task itself is just a utility, we are using this as a tool to deploy at the moment but it may change in the future.